src/ocaml/merlin_specific/browse_raw.ml

```diff
@@ -346,48 +340,14 @@ let of_method_call obj meth loc =
   let loc = {loc with Location. loc_start; loc_end} in
   app (Method_call (obj,meth,loc)) env f acc
 
-<<<<<<< HEAD
-let of_function_param (param : Typedtree.function_param) =
-  (* We should consider taking into account param.fp_loc at some point, as it
-     allows us to respond with the *parameter*'s type (as opposed to the
-     function's type) when the user queries the label:
-
-     let f ?y:(x = 3) () = x
-           ^
-  *)
-  match param.fp_kind with
-  | Tparam_pat pat -> of_pattern pat
-  | Tparam_optional_default (pat, expr, _) ->
-      of_pattern pat ** of_expression expr
```

Comment:
I don't know the story behind this change - it looks like you made this a few months before I started - but it looks like you didn't preserve it. Was that intentional?

If the answer is yes, I trust that it's fine. I just want to make sure it wasn't a mistake.

Reply:
The code appears elsewhere in the file. Keeping it was intentional:

```ocaml
(* We should consider taking into account param.fp_loc at some point, as it
   allows us to respond with the *parameter*'s type (as opposed to the
   function's type) when the user queries the label:

   let f ?y:(x = 3) () = x
         ^
*)
and of_function_param fp = of_function_param_kind fp.fp_kind

and of_function_param_kind = function
  | Tparam_pat pat -> of_pattern pat
  | Tparam_optional_default (pat, exp, _) ->
    of_pattern pat ** of_expression exp
```

Reply:
Ah I missed that. Looks good then